EveHQ.Prism/Classes/StructurePreference.vb

```diff
@@ -0,0 +1,113 @@
+﻿Imports EveHQ.EveData
+
+''' <summary>
+''' A Structure Definition
+''' </summary>
+Public Class StructurePreference
+
+    ''' <summary>
+    ''' The Name of the Structure
+    ''' </summary>
+    Private _name As String
+
+    ''' <summary>
+    ''' The Name of the Structure
+    ''' </summary>
+    Public Property Name() As String
+        Get
+            Return _name
+        End Get
+        Set(value As String)
+            _name = value
+        End Set
+    End Property
+
+    ''' <summary>
+    ''' The Type of the Structure
+    ''' </summary>
+    Private _type As EveType
+
+    ''' <summary>
+    ''' The Type of the Structure
+    ''' </summary>
+    Public Property Type() As EveType
```

Comment:
Well, that's already structure, so `StructureItemType` doesn't make sense.
`ItemType` ?